### DISCUSSION TITLE
Large memory nodes

### DISCUSSION THREAD

**Discussion by mflynn-lanl** [microbiomedata/nmdc-edge#287] [General]
@edwins We had discussed using large memory nodes at the beginning of the project. If I recall correctly, there are large memory VMs that we can use, but I don't know how to create them. They're not an option available in exosphere, horizor or cacao

**Comment by edwins** [comment]
@mflynn-lanl the large memory nodes are technically a separate resource. You'll need to allocate credits to use them. As an example:

![image](https://github.com/user-attachments/assets/2c9555bf-fa1a-4d28-8987-5c2bbe096dd8)
